src/py/flwr/server/superlink/driver/driver_servicer.py

```diff
@@ -208,13 +204,47 @@ def PullServerAppInputs(
         self, request: PullServerAppInputsRequest, context: grpc.ServicerContext
     ) -> PullServerAppInputsResponse:
         """Pull ServerApp process inputs."""
-        raise NotImplementedError()
+        log(DEBUG, "DriverServicer.PullServerAppInputs")
+        state = self.state_factory.state()
+        ffs = self.ffs_factory.ffs()
+
```

Comment:
let's add a single line comment of what lines below do?

Reply:
Sure!